### ISSUE TITLE
Add support for MariaDB LTS versions

### ISSUE THREAD

**Issue by cngJo** [ddev/ddev#4894]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Not really an issue, but I noticed that currently the highest supported mariadb version is 10.8 whilst the latest mariadb is actually 10.11.

### Describe your solution

Add database containers for MariaDB 10.9, 10.10, 10.11

### Additional context

- f33e466dcac5a3ac652ce2b4e3b5818a96e39652 added support for MariadDB 10.8
- MariaDB Version list: https://mariadb.org/mariadb/all-releases/

**Comment by rfay** [comment]
I'm interested in your request. I had been trying to keep up with mariadb versions, but they came so fast and furious, and had short lifetimes, and nobody seemed to use the newer ones. 10.8 and 10.9 are almost EOL already.  10.11 looks like it's really worth doing (LTS). Probably we should just do LTS versions in the future?

And then there's 11.0 and 11.1. What will the next LTS in the 11.x series be? It's not clear. 

* https://mariadb.org/about/#maintenance-policy
* https://endoflife.date/mariadb

Interested in your feedback. I think the most likely thing is to support new LTS releases. With the current architecture for mariadb/mysql a docker image is built for each version, which takes time and slows down the image build. (PostgreSQL instead uses the upstream image and just adds a layer at `ddev start` time, so it's less costly.)

**Comment by cngJo** [comment]
Thanks for your feedback!

I get the point that supporting all versions just spikes build times for little to no effort.

I stumbled across this, wanting to update my MariaDB to 10.11 for a planned DB Update in a project.
I've seen that all MariaDB 10.x-10.8 are supported, that's why I phrased the like for supporting all versions.

In my opinion, supporting "only" the LTS versions in the future would be a good solution here. 

Regarding MariaDB 11, It seems like the first versions (11.0 / 11.1) will not be a LTS release, I guess that will come in the future but not planned for low minor releases. 
(Which makes sense, to not release 11.0 as LTS which little to no experience running it in prod - IMHO) 

I'd try to create a Pull Request the next days, to add support for MariaDB 10.11. 
Am I right that I mostly can copy the `10.8` parts of https://github.com/ddev/ddev/commit/f33e466dcac5a3ac652ce2b4e3b5818a96e39652?
